.travis.yml

```diff
@@ -59,15 +59,56 @@ jobs:
     # ---------------------------------------
     # Regular Build & Deploy
     # ---------------------------------------
-    # console.kuzzle.io
+    # console.kuzzle.io/kuzzle-v1
```

Comment:
Sorry, why don't we take this out of the v3 branches?

Reply:
For now I just wanted to fix what was broken without touching the travis.yml too much.

Reply:
I understand... I'd be confident, since the PR points on the dev branch and, in the worst case, the damage is done on next-console, what do you think?

Reply:
yes, I agree ! I will clean up this travis file ;)